Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -190,13 +196,28 @@ final class RewardsCollectionViewController: UICollectionViewController {
     return CGSize(width: itemWidth, height: itemHeight)
   }
 
+  private func goToPledge(project: Project, reward: Reward, refTag: RefTag?) {
```

Comment:
The `refTag` argument is not being used here. Could you remove it from the function signature?